app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,11 @@
 <resources>
     <string name="app_name">Youthero</string>
+    <string name="education">Educatie</string>
```

Comment:
Need to have a better convension for naming this resources in a bigger app, usually is `screeName_fieldName`

Reply:
Thank you for info